src/content/en/2024/structured-data.md

```diff
@@ -2,17 +2,611 @@
+
+## AI, Machine Learning, and Structured Data
+
+The synergy between structured data and AI/ML has deepened. Structured data is crucial in training machine learning models, providing consistent, machine-readable labels. It's particularly important in areas such as:
+
+* **Large Language Models (LLMs)**: Fine-tuning with structured data for improved performance in specific domains.
+* **Explainable AI**: Using knowledge graphs to trace and explain AI decision-making processes.
+* **Multimodal AI**: Linking different data types (text, images, video) in AI systems.
+
+## Semantic SEO and Data Quality
+
+Semantic SEO has evolved from focusing purely on keyword matching to leveraging structured data for deeper contextual understanding, which is now essential for search engines to provide more accurate and relevant results. It allows websites to build thematic depth and meaning into their content by using entities and structured metadata, enhancing the ability of search engines like Google and Bing to grasp the intent behind queries rather than just focusing on keyword frequency.
+
+By implementing semantic SEO, businesses can create content clusters based on topics, not just individual keywords, making their content more discoverable and contextually relevant across various search platforms, including voice search assistants. This approach significantly boosts search engine rankings and user engagement, as structured data allows search engines to understand the content at a more granular level, making it easier to match user intent.
+
+Data quality plays a key role here as well. High-quality structured data ensures consistency and accuracy, which is crucial not only for search engines but also in combating misinformation. It helps maintain trustworthiness across the web, especially as structured data is increasingly used in AI-powered systems like knowledge graphs for fact validation and enhancing large language model (LLM) training.
+
+For example, organizations such as EssilorLuxottica, Shiseido and others are using semantic technologies like knowledge graphs to link content and provide users with more detailed, contextually relevant results. This practice also aids in AI-powered content discovery and makes content easier to retrieve through Generative Search like Perplexity or You.com.
+
+Investing in semantic SEO and maintaining high-quality structured data not only enhances search visibility but also lays a foundation for future-proofing content for AI-driven discovery.​
+
+## A Year In Review
+
+## Structured Data
+
+The landscape of structured data implementation continues to evolve, with RDFa and Open Graph maintaining their dominant positions, now reaching 66% and 64% of pages respectively. X (Twitter) meta tags have shown significant growth, appearing on 45% of pages, while JSON-LD usage has expanded to 41% of pages, reflecting its growing adoption for structured data implementation.
```

Comment:
One of my big gripes with this report over the years has been the conflation of syntax/encoding with schema.  OGP is actually RDFa.  It's a vocabulary of RDFa and even specifies a RDF schema.  You could encode schema.org this way (and it's weirdly possible some of Google systems would read it, we just convert them to triples either way).  Similarly, you could put OGP in JSON-LD.  Yes, most consumers probably wouldn't read it, but it's orthogonal technically.  Dublin core can be encoded several ways but you're probably talking about RDFa as well.

Things that I think are syntax/encoding:
JSON-LD
Microdata
RDFa
HEAD data (non RDFa meta tag usage like Twitter cards)
Class metadata (if you want to give a name to whatever microformats does)

Things that I feel are vocabulary:
schema.org
OGP
Twitter
dublin core
Microformats

I'm aware that this has been done like this for awhile and so there is value in some consistency with regards to over-time graphs.  And I agree it would be silly to not recognize that no one is using class metadata encoding except microformats so maybe that doesn't need to be discussed separately.

But this text often compares them side by side and that's the part I feel like can be shifted a bit.

Reply:
I have added some details to more clearly differentiate syntax/encoding.